CMakeLists.txt

```diff
@@ -162,6 +186,38 @@ endif()
 
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
+
+# *****************************************************************************
+# Dependency // SIMDOperators
+# *****************************************************************************
+include(FetchContent)
+
+FetchContent_Declare(
+  SIMDOperators
+  GIT_REPOSITORY https://github.com/db-tu-dresden/SIMDOperators.git
+  GIT_TAG        daphne_integration # or a later release
+)
+
+FetchContent_MakeAvailable(SIMDOperators)
```

Comment:
FetchContent alone did not set the simd ops up properly for me - maybe try to make it work in a daphne_dev container?